pkg/controller/releasemanager/syncer.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/go-logr/logr"
```

Comment:
This is officially the worst part of picchu! All who enter beware.

Reply:
🤣